sagemaker-training-compiler/huggingface/tensorflow_multiple_gpu_multiple_node/scripts/train_with_keras.py

```diff
@@ -0,0 +1,151 @@
+import argparse
+import logging
+import os
+import sys
+import json
+
+import tensorflow as tf
+from datasets import load_dataset
+from transformers import AutoTokenizer, TFAutoModelForSequenceClassification
+
+from tensorflow.keras import mixed_precision
+
+# Parse sagemaker tensorflow distributed configuration
```

Comment:
SageMaker Training ToolKit already seems to be doing this: https://github.com/aws/sagemaker-tensorflow-training-toolkit/blob/master/src/sagemaker_tensorflow_container/training.py#L37. 

Please provide differentiation and justification.